### PR TITLE
Support for vulnerability scanner files

### PR DESCRIPTION
**Pull request by mikaelforeseeti**

### CODE REVIEW

File: securicad/vanguard/client.py [hunk 1]
```diff
@@ -100,7 +118,10 @@ def model_request(self, model_tag):
         data = {"mtag": model_tag}
         res = requests.post(url, headers=self.headers, json=data)
         res.raise_for_status()
-        return res.status_code, res.json()["response"]
+        if res.status_code == 204:
+            return res.status_code, {}
+        else:
+            return res.status_code, res.json()["response"]
```
Comment (maxwalls) [inline securicad/vanguard/client.py:124]: Shouldn't this also be done in `simulate_model`?

File: securicad/vanguard/client.py [hunk 2]
```diff
@@ -45,10 +45,10 @@ def simulate(self, model, profile):
 
     def get_model(self, **kwargs):
         if kwargs.get("data"):
-            model_tag = self.build_from_config(kwargs.get("data"))
+            model_tag = self.build_from_config(kwargs.get("data"), kwargs.get("vuln_data"))
         else:
             model_tag = self.build_from_role(
-                kwargs.get("access_key"), kwargs.get("secret_key"), kwargs.get("region")
+                kwargs.get("access_key"), kwargs.get("secret_key"), kwargs.get("region"), kwargs.get("vuln_data"),
```
Comment (maxwalls) [inline securicad/vanguard/client.py:51]: Mention this new arg in `README.md`

File: securicad/vanguard/client.py (hunk 1 above)
Comment (mikaelforeseeti) [reply securicad/vanguard/client.py:124]: I added it to get_results

Comment (maxwalls) [reply securicad/vanguard/client.py:124]: How about `model_request`?

File: README.md [hunk 3]
```diff
@@ -76,6 +76,9 @@ The results will be returned as a `dict` with your high value asset identifiers
 ```
 Check out `example.py` for a more detailed example.
 
+#### Vulnerability scan results
+Ifyou have a vulnerability scan result file you can attach it when making the call to get_model, keyword `vuln_data`. See `example.py` for details.
```
Comment (eringdahl) [inline README.md:80]: space

File: README.md [hunk 4]
```diff
@@ -76,6 +76,9 @@ The results will be returned as a `dict` with your high value asset identifiers
...
+
```
Comment (maxwalls) [inline README.md:81]: `example.py` does not contain anything related to this.

Comment (maxwalls) [reply README.md:81]: Suggestion:
```
If you have vulnerability scanner data in a JSON file, you can pass it to the `client.get_model()` call in both examples above:

with open('vuln_data.json', mode='r', encoding='utf-8') as vuln_file:
    vuln_data = json.load(vuln_file)
model = client.get_model(data=data, vuln_data=vuln_data)
```

**Pull request merged**